spec/post_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is not valid without a title' do
+    user = User.create(name: 'Jane Smith', posts_counter: 2)
+    post = user.posts.new(title: nil, comments_counter: 1, likes_counter: 0)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid with a long title' do
+    user = User.create(name: 'Alice Johnson', posts_counter: 3)
+    post = user.posts.new(title: 'A' * 251, comments_counter: 1, likes_counter: 0)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid with a negative comments_counter' do
+    user = User.create(name: 'Bob Smith', posts_counter: 1)
+    post = user.posts.new(title: 'Sample Post', comments_counter: -1, likes_counter: 0)
+    expect(post).to_not be_valid
+  end
+
+  it 'is valid with zero comments_counter' do
+    user = User.create(name: 'Eve Johnson', posts_counter: 4)
+    post = user.posts.new(title: 'Sample Post', comments_counter: 0, likes_counter: 2)
+    expect(post).to be_valid
+  end
```

Comment:
- [ ] Good job for testing the `comments_counter` validation 👏However, as per [this project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations), we need to test all the validations, therefore, kindly add unit tests for the validation `LikesCounter must be an integer greater than or equal to zero.`